### DISCUSSION TITLE
Motion detection configurable area

### DISCUSSION THREAD

**Discussion by base1983** [Sunoo/homebridge-camera-ffmpeg#1293] [Q&A] [answered]
I got it working with mqtt sending me a popup on my iphone, but it also shows cars passing by. Can I somehow filter that only close to the front door motion triggers a notification?

**Comment by Sunoo** [comment, marked as answer]
What are you using to trigger motion? This plugin just passes motion alerts from other things along to HomeKit.

That said, HKSV should be out shortly, and my understanding is that can help filter things.

**Reply by base1983** [reply]
I used:

https://sunoo.github.io/homebridge-camera-ffmpeg/automation/motion.html
